cmd/minersc.go

```diff
@@ -264,12 +291,30 @@ var minerscSharders = &cobra.Command{
 		if allFlag {
 			sharders := new(zcncore.MinerSCNodes)
 			callback := NewJSONInfoCB(sharders)
-			if err = zcncore.GetSharders(callback); err != nil {
-				log.Fatalf("Failed to get registered sharders: %v", err)
+
+			limit, offset := 20, 0
+			active := false
+			if flags.Changed("limit") {
+				limit, err = flags.GetInt("limit")
+				if err != nil {
+					log.Fatal(err)
+				}
+			}
+
+			if flags.Changed("offset") {
+				offset, err = flags.GetInt("offset")
+				if err != nil {
+					log.Fatal(err)
+				}
 			}
-			if err = callback.Waiting(); err != nil {
-				log.Fatalf("Failed to get registered sharders: %v", err)
+
+			if flags.Changed("active") {
+				active, err = flags.GetBool("active")
+				if err != nil {
+					log.Fatal(err)
+				}
 			}
+			zcncore.GetSharders(callback, limit, offset, !active)
```

Comment:
Shouldn't we use `active` rather than `!active`?

Reply:
@peterlimg 
```go
wg.Add(1)
zcncore.GetBlobbers(statusBar, limit, offset, !active)
```

In blobbers also, we used !active, so decided to follow that

Reply:
Hey @peterlimg, I have changed the usage in code itself from `!active` to `active`.
Currently, now if u run `zwalletcli ls-miners` if there are no active miners u will get an empty list by default.
one need to explicitly pass `--active` to get miners.